pages/components/themeComponents/ThemeProvider.js

```diff
@@ -12,11 +11,32 @@ const ThemeContext = React.createContext(defaultContextData);
 const currentThemeState = () => React.useContext(ThemeContext);
```

Comment:
This is essentially a hook since it is a function that uses a hook.

Hooks should be prefixed with `use`, to provide the context that it is actually a hook.

Something like `ùseCurrentTheme`, or just `useTheme` would be ok.